software/kernel/signpost_drivers/src/gps_console.rs

```diff
@@ -162,10 +151,11 @@ impl<'a, U: UARTAdvanced> Driver for Console<'a, U> {
                     // only both receiving if we've got somewhere to put it
                     app.read_buffer = app.read_buffer.take().map(|app_buf| {
                         self.rx_buffer.take().map(|buffer| {
+                            /*
                             let mut receive_len = app_buf.len();
                             if receive_len > buffer.len() {
                                 receive_len = buffer.len();
-                            }
+                            }*/
```

Comment:
Yeah, I removed warnings from the gps_console while I was at it. receive_len is unused so I just added it to the commented out mess that is the gps_console